contributing.md

```diff
@@ -47,12 +51,6 @@ If the manatainer or owner finds any blank issue or any kind of unethical behavi
     "Twitter": "YOUR_TWITTER_ACCOUNT_LINK"
   },
   "Projects": [
-    {
-      "link": "PROJECT_LINK",
-      "title": "PROJECT_NAME",
-      "description": "PROJECT_DESCRIPTION",
-      "tech": ["tech1", "tech2"]
-    },
```

Comment:
add this sample code so that users can understand he/she can add more projects here.

```
    {
      "link": "PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    }
```